agent/js/README.md

```diff
@@ -18,12 +21,12 @@ import Agent, {
 } from "agent-protocol";
 
 const taskHandler = async (
```

Comment:
I generally prefer the functions to be declared as
```
function foo() {}
```
and not anonymous functions assigned to a variable, but this is a minor thing.

Reply:
I can change that in the example if you'd like.

Reply:
I changed it in the example and readme.